pep_sphinx_extensions/pep_zero_generator/writer.py

```diff
@@ -177,9 +177,10 @@ def write_pep0(self, peps: list[PEP], header: str = HEADER, intro: str = INTRO,
         # PEP types key
         self.emit_title("PEP Types Key")
         for type_ in sorted(TYPE_VALUES):
-            self.emit_text(f"    {type_[0]} - {type_} PEP")
+            self.emit_text(f"* {type_[0]} - {type_} PEP")
             self.emit_newline()
 
+        self.emit_text("More info in :pep:`PEP 1 <1#pep-types>`.")
```

Comment:
You made three other suggestions, and I chose the first :)

> I'm not sure the section name is needed at all, since you're linking directly to it. You could just say "More info in [PEP 1](https://github.com/python/peps/pull/LINK)" or to clarify both links aren't the same and just to the top level of PEP 1, "See [the descriptions in PEP 1](https://github.com/python/peps/pull/LINK)". Or even just "[More info](https://github.com/python/peps/pull/LINK)" for both.

But happy to go with this one :)

Reply:
Oh sorry! Seems like I was the one who got lost, doh!